commerce-checkout-web/src/main/resources/META-INF/resources/checkout_step/address.jsp

```diff
@@ -123,38 +123,24 @@ long commerceRegionId = BeanParamUtil.getLong(currentCommerceAddress, request, "
 	Liferay.provide(
 		window,
 		'<portlet:namespace />addStreetAddress',
-		function() {
-			var A = AUI();
+		function <portlet:namespace />addStreetAddress() {
+			const A = AUI();
```

Comment:
You seem to be removing a lot of newlines.... some of them were intended to be meaningful. I'll try to find a reference for Liferay newline rules -- I think they're the same in JS and Java.

Reply:
Here's a reference for Java at least: https://docs.google.com/document/d/188YFP0V4n9WJIPYZL3jGBRBvEsvBNmHOU97Fdd_g6Ew/edit#heading=h.e66kbvnr90es.